www_src/lib/platform.js

```diff
@@ -0,0 +1,171 @@
+/**
+ * Device agnostic platform interface.
+ *
+ * @package core
+ * @author Andrew Sliwinski <a@mozillafoundation.org>
+ */
+
+var lru = require('lru-cache');
+
+/**
+ * Constructor
+ */
+function Platform () {
+  // Check to see if window.Platform exists. If it does, return it.
```

Comment:
As per the description, this checks to see if the platform (in our current case, Android) was bound an interface to `window.Platform`. If it has, the platform module returns it instead of the polyfill.